### PR TITLE
fix windows compile errors

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

File: src/vtkh/StatisticsDB.hpp [hunk 1]
```diff
@@ -73,9 +73,8 @@ class VTKH_API EventHistory
 #ifdef VTKH_PARALLEL
         return MPI_Wtime();
 #else
-        struct TIMEINFO ti;
-        gettimeofday(&ti, 0);
-        return (double)(ti.tv_sec + ti.tv_usec/1e6f);
+    return std::chrono::system_clock::now().time_since_epoch() /
```
Comment (cyrush) [inline src/vtkh/StatisticsDB.hpp:76]: @biagas  This should should suffice  as sub for the old logic on windows. Can you give it a try?

Comment (cyrush) [reply src/vtkh/StatisticsDB.hpp:76]: (I pushed this to your branch and opened a PR)

Comment (biagas) [reply src/vtkh/StatisticsDB.hpp:76]: Thanks, I will check it out.

Comment (biagas) [reply src/vtkh/StatisticsDB.hpp:76]: This branch compiles on Windows. Thanks @cyrush

**Review by biagas — APPROVED** [review]

**Pull request merged**